docs/handbook/systems-administration.md

```diff
@@ -293,6 +293,178 @@ Now user whoever can shutdown the system.
 The `pfexec` command is more flexible in the number of privileges that can be
 assigned to a user.
 
+
+## [Software management](#software-management)
```

Comment:
The link target to this section is automatically generated by mkdocs, as with all sections. This is therefore a link to itself and should be removed.

Reply:
@JMadgwick 
`pfexec` is still all todo. I' not entirely sure as to what your question is

Reply:
The link markdown here does nothing, it doesn't create a link target, it is not required. It should be:
```suggestion
## Software management
```

Reply:
Again, please bear with me: the original doc that I wrote just simply kept growing. I had one single, large document, which I had to sub-divide into smaller pieces for the PRs.
So the link you are referring to is in another branch, in fact, the other PR232 I think. So yes, the link is _currently_ broken, but as soon all PRs are integrated, the links should work.
It was simply easier for me to do it that way.

You might prefer to create an issue and assign it to me, and I'll resolve later

Reply:
Ok, I misunderstood the purpose of the link. So this heading will then link to another page or section? I'm not sure that headings should be used for linking in that way.

I've experienced the situation of multiple smaller PRs instead of one big PR before. I've always thought it easier to have a big PR as it avoids dependencies between PRs which can cause confusion - as happened here.

I don't think there's a rush to get this merged in. You could mark PRs as draft if you're still working on the commits. I'll wait before picking out any minor issues, I'd just assumed this link was a mistake.

Reply:
Think of it like this, for each PR, I'd have to _remove_  all internal links for each PR (that would become broken) and then _not to forget_ to reinstall later. There are a couple of procedures for doing this; but I did I selected the easiest for me: the links will fit like a nice jigsaw later and nothing is forgotten. However PR should always work in their own right, and strictly speaking the links to work: they just do nothing!

What you might prefer to do---as you noticed something awry with the PR and however well my promises might be you'd still prefer insurance that the issue will indeed be corrected and not forgotten---is to create an issue and directly assign it to me. I have absolutely no problem with this.

Reply:
@JMadgwick 
You generally want PRs to be as small as possible, self-contained and being coherent as in being a single unit.

Big PRs are very difficult to review.  Even the PR I just created _SMF_ is too big, although it encompasses only one single section. I've got one to come which is much too big: System Monitoring.

You are correct, there is no hurry to get these merged; but they have been hanging about for at least a year as I hadn't any time to complete them. The notes might be beneficial to others, and others might be able to _fill in the gaps_; so a draft isn't really going to help. Other can, for example, add their own recipes for pkg